src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -368,7 +368,7 @@ export function CollapsibleNavGroupEnabled({
   return (
     <>
       <EuiHideFor sizes={['xs', 's', 'm']}>{rendeLeftNav()}</EuiHideFor>
-      <EuiHideFor sizes={['xl', 'l']}>
+      <EuiHideFor sizes={['xl', 'l', 'xxl', 'xxxl']}>
```

Comment:
maybe changed to showFor xs, s and m to avoid we have more xxxxxL

Reply:
Good idea, done for that.